modules/ztp-generating-ran-policies.adoc

```diff
@@ -8,23 +8,11 @@
 .Prerequisites
 
 * Install Kustomize
-* Install golang
+* Install the Kustomize Policy generator plug-in
```

Comment:
IMHO 
Use Kustomize to install policy generator.

Reply:
I followed the info in https://github.com/stolostron/policy-generator-plugin
it didn't seem to use anything in particular to install the plug-in, (I mean, basically you copy the binary in the right place) which is why I worded it that way.

Other plugins I found are installed the same way, and I couldn't find a way to specifically ask Kustomize to install a plugin -- but maybe I didn't find the right docs?

Thanks!

Reply:
Chatting over slack. Also, added link to the plug-in.